index.js

```diff
@@ -21,16 +21,22 @@ var loop = function () {
       log.info('Couch2pg Migration checks complete');
     })
     .then(couch2pg.import)
-    .then(function () {
+    .then(function(summary) {
       log.info('Imported successfully at ' + Date());
-    })
-    .then(xmlforms.migrate)
-    .then(function() {
-      log.info('xmlforms Migration checks complete');
-    })
-    .then(xmlforms.extract)
-    .then(function () {
-      log.info('XML forms completed at ' + Date());
+
+      if (summary) {
```

Comment:
I feel like there should always be a summary even if it's `{ edited: [], deleted: [] }`.